tests/virt/node/descheduler/conftest.py

```
@@ -519,3 +489,36 @@ def utilization_imbalance(
             pod_prefix=utilization_imbalance_deployment_name,
         )
     )
+
+
+@pytest.fixture(scope="class")
+def node_to_run_stress(schedulable_nodes, deployed_vms_for_descheduler_test):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)